libs/components/src/progress-bar/vl-progress-bar.component.ts

```diff
@@ -86,12 +86,14 @@ export class VlProgressBarComponent extends BaseLitElement {
                 aria-label=${step}
                 id="step-${index + 1}"
             >
-        
-                <vl-popover for="step-${index + 1}" placement="top" trigger=${isPopOverActive
-                    ? 'focus hover'
-                    : undefined}>${isPopOverActive ? step : undefined}</vl-popover>
+                <vl-popover
+                    for="step-${index + 1}"
+                    placement="top"
+                    trigger=${isPopOverActive ? 'focus hover' : undefined}
+                    >${isPopOverActive ? step : undefined}</vl-popover
+                >
```

Comment:
Het probleem was er al - we hebben dit over het hoofd gezien in een vorige PR - maar ik zie dat bij "hover" we een lege popover te zien als `showLabels` op `true` staat:

<img width="160" alt="image" src="https://github.com/milieuinfo/uigov-web-components/assets/15731348/bb0d3a8c-6180-4a86-9bf9-be9a5d54b4bc">

Eventuele fix zodat popover helemaal niet te zien is:

```ts
 ${isPopOverActive
? html`
    <vl-popover for="step-${index + 1}" placement="top" trigger="focus hover">
        ${step}
    </vl-popover>
        ` : nothing}
```

Reply:
Voorgestelde fix toegepast.